src/adaptors/graphql/index.js

```diff
@@ -0,0 +1,82 @@
+import { createStore, combineReducers } from 'redux';
+import { graphql } from 'graphql';
+
+import parseSchema from './utils/parseSchema';
+import normalize from './utils/normalize';
+import merge from './utils/merge';
+
+const UPDATE_CACHE = 'UPDATE_CACHE';
+
+export default function createAdaptor(endpoint, schema) {
+  const parsedSchema = parseSchema(schema);
+  const reducers = Object.keys(parsedSchema).reduce((acc, key) => {
+    return {
+      ...acc,
+      [key]: createReducer(key),
+    };
+  });
+  const reducer = combineReducers(reducers);
+  const store = createStore(reducer);
```

Comment:
I'm not quite sure about how to it right but I think about exporting all the stuff from graphql adaptor implementation so everyone can do some custom logic utilizing my work on this and using this as a referrence implementation. So probably one can construct the store in a way he like or even not use redux at all. The point here is I do not think we can provide a really general interface which can cover all the edge cases.